### PR TITLE
Arrays Lab

### PR DESCRIPTION
**Pull request by gurliaccio**
{
comfort: 4
completeness: 5
}

### CODE REVIEW

**Comment by rgpass** [comment]
Be succinct in your commit messages.

File: my_talkative_robot.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+puts "My shopping list:"
+grocery_list = ["cashews", "juice", "jerky", "crackers", "milk"]
+puts grocery_list
+user_questions = {}
+user_questions = { nuts: "cashews", beverage: "juice", dinner: "jerky", lunch: "crackers",
```
Comment (rgpass) [inline my_talkative_robot.rb]: This should read like

``` ruby
user = []
user[:age] = get_user_age
user[:gender] = get_user_gender
```

Perhaps you meant to name this hash something like favorite_foods?

File: my_talkative_robot.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+user_questions = {}
+user_questions = { nuts: "cashews", beverage: "juice", dinner: "jerky", lunch: "crackers", 
+				   nightcap: "milk" }
+thing = grocery_list[1]
+puts "Did you already grab the #{thing}?"
+grocery_list.delete(thing)
+puts grocery_list
+puts "Oh shit, don't forget the bread!"
+grocery_list << "bread"
+puts grocery_list
```
Comment (rgpass) [inline my_talkative_robot.rb]: This lab should add onto the previous lab. Bring all your previous stuff back in here.

**Comment by rgpass** [comment]
Once you get a solid `user` hash, I'll approve. When you push up the changes, put a comment of @rgpass so I get the notification.

**Comment by gurliaccio** [comment]
Marc Merged

**Pull request merged**